src/blog/2023/01/getting-started-with-node-red.md

```diff
@@ -9,9 +9,9 @@ tags:
     - posts
     - node-red
     - how-to
-    - Node-RED getting started
-    - Node-RED Tutorial
-    - Install Node-RED
+    - nodered getting started
```

Comment:
Because nobody searches 'Node-RED' exactly as written, we, as a company, ensure to write it this way for branding. However, when people search on Google, they often use variations like 'nodered' or 'node red' all time.
Below is the google keyword planner result

![Screenshot 2024-07-20 181057](https://github.com/user-attachments/assets/9fc4a22f-f167-4f43-a377-395c8e6ff4ab)